setup.cfg

```diff
@@ -10,6 +10,9 @@ oteapi.mapping =
 oteapi.filter =
   oteapi.filter/crop = oteapi.strategies.filter.crop_filter:CropImageFilter
   oteapi.filter/sql = oteapi.strategies.filter.sql_query_filter:SQLQueryFilter
+oteapi.resource =
+ oteapi.resource/vnd.postgres = oteapi.strategies.resource.application_vnd_postgres:PostgresResourceConfig
```

Comment:
vnd.postgres in not a valid mime-type and doesn't make sence here. Rename to oteapi.resource/postgresql

Reply:
Done